Source/Diagnostics/ReducedDiags/FieldProbe.cpp

```diff
@@ -34,7 +34,9 @@
 #include <ostream>
 #include <string>
 #include <unordered_map>
-
+#include <vector>
+#include <iostream>
+#include <stdlib.h>
```

Comment:
The equivalent C++ include is:
```suggestion
#include <cstdlib>

```
Ref.: https://en.cppreference.com/w/cpp/header/cstdlib